### PR TITLE
chore: Run MS Docs "Generat Master Redirect"

### PR DESCRIPTION
**Pull request by nschonni**
- Autoformated openpublishing.redirect
- Replaces files with "redirect_url" with redirect entry
- Manually removed "docs.microsoft.com" from redirect URLs to site absolute

### CODE REVIEW

File: education/windows/create-tests-using-microsoft-forms.md [hunk 1]
```diff
@@ -1,33 +0,0 @@
...
-manager: dansimp
-description: Learn how to use Microsoft Forms with the Take a Test app to prevent access to other computers or online resources while completing a test.
-keywords: school, Take a Test, Microsoft Forms
-ms.prod: w10
-ms.mktglfcycl: plan
-ms.sitesec: library
-ms.pagetype: edu
-author: levinec
-ms.author: ellevin
-redirect_url: https://support.microsoft.com/help/4000711/windows-10-create-tests-using-microsoft-forms
```
Comment (nschonni) [inline education/windows/create-tests-using-microsoft-forms.md]: Not sure if this was intentional, but because of the metadata, the file was redirecting

**Comment by nschonni** [comment]
Because of the reformatting of the openpublish.redirect.json, it's easier to use the "ignore whitespace" option in the diff view

**Comment by e0i** [comment]
@levinec @mjcaparas @DaniHalfin @Dansimp 

The PR that introduces automated changes regarding URL redirects is available for review.

**Comment by levinec** [comment]
@e0i once conflicts are resolved, I will :)

**Pull request merged**